sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/simple/RegressionTests.java

```diff
@@ -1496,11 +1496,6 @@ public void testSlt() {
 
     @Test
     public void issue3199() {
-        this.statementsFailingInCompilation("""
```

Comment:
Is this related to the PR?

Reply:
it's a bug fixed in Calcite which makes this test wrong.